composer.json

```diff
@@ -51,10 +51,14 @@
         "smarty/smarty": "^3.1",
         "james-heinrich/phpthumb": "^1.7",
         "erusev/parsedown": "^1.7",
-        "pelago/emogrifier": "^2.0"
+        "pelago/emogrifier": "^2.0",
+        "simplepie/simplepie": "^1.5"
     },
     "autoload": {
-        "classmap": ["core/xpdo/"]
+      "classmap": [
+        "core/xpdo/",
+        "core/model/modx"
```

Comment:
is this necessary here?

Reply:
I've experimented with external core and found out that MODX classes not added to autoload configuration of the composer. So maybe this PR not the best place for this change, but it should be in MODX 3 to get MODX core classes available via autoload.php.